engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -389,6 +375,12 @@ def scc_upgrade
           auth = request.headers.fetch('HTTP_AUTHORIZATION', '')
           SccProxy.scc_upgrade(auth, @product, @system, logger)
           logger.info "System #{@system.login} successfully upgraded with SCC"
+          InstanceVerification.update_cache(
+            InstanceVerification.build_cache_entry(
+              request.remote_ip, @system.login, @system.pubcloud_reg_code, @system.proxy_byos_mode, @product
+            ),
```

Comment:
Here we are just passing the product object, which is incompatible with the change in #1318 . I think passing the product is sufficient, see comments in #1318

Reply:
it is not, this PR only pass the product because the change to fix that is implemented in a different PR
not this PR responsibility